content/source/docs/enterprise/workspaces/creating.html.md

```diff
@@ -39,12 +40,12 @@ VCS-backed workspaces support several optional fields, which you can reveal by c
 
 - **Terraform working directory** — The directory where Terraform will execute, specified as a relative path from the root of the repo. This is useful when working with VCS repos that contain multiple Terraform configurations. Defaults to the root of the repo.
 
-    -> **Note:** If you specify a working directory, TFE will still queue a plan for changes to the repository outside that working directory. This is because local modules are often outside the working directory, and changes to those modules should result in a new run. If you have a repo that manages multiple infrastructure components with different lifecycles and are experiencing too many runs, we recommend splitting the components out into independent repos. See [Repository Structure](./repo-structure.html) for more detailed explanations.
+  -> **Note:** If you specify a working directory, TFE will still queue a plan for changes to the repository outside that working directory. This is because local modules are often outside the working directory, and changes to those modules should result in a new run. If you have a repo that manages multiple infrastructure components with different lifecycles and are experiencing too many runs, we recommend splitting the components out into independent repos. See [Repository Structure](./repo-structure.html) for more detailed explanations.
```

Comment:
@mikenomitch I think we'd be better to mention filtering here, at this point!

Reply:
Heh, I didn't see this at all (only in the diff cause of a linter change), but I'll add something about it!

Reply:
FYI, I spoke with the rest of the team and they'll handle this in a different PR/ticket.